### DISCUSSION TITLE
Map Picker scaling issue when Hires enabled

### DISCUSSION THREAD

**Discussion by jaalibandar** [cables-gl/cables_docs#181] [! Welcome - readme first!]
Link to test patch: https://cables.gl/edit/jO9MiY 

The picker works fine normally - however, when I turn on the hires button in mainloop - the scaling seems a bit off. 
As discussed in the last meetup it seems to be a result of the display size being set to 125%.

**Comment by pandrr** [comment]
thx for the report. 
i fixed it on dev.cables.gl please test there

**Comment by pandrr** [comment]
no problem, you are welcome!
